src/controllers/dynakube/version/updater.go

```diff
@@ -179,6 +190,10 @@ func updateVersionStatusForTenantRegistry( //nolint:revive
 	return nil
 }
 
+func isOneAgentImage(imageID string) bool {
+	return strings.Contains(imageID, "oneagent")
+}
```

Comment:
that is rather a weak check also the "design" of this was meant have these specific checks in each `Updaters` (the oneagent `src/controllers/dynakube/version/oneagent.go`)

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/2156/commits/72e5f15c7b6a88a9ca8a6cc78498154bd11d4244